test.sh

```diff
@@ -250,16 +276,20 @@ docker run --rm \
     -e DCOS_LOGIN_PASSWORD="$DCOS_LOGIN_PASSWORD" \
     -e CLUSTER_URL="$CLUSTER_URL" \
     -e S3_BUCKET="$S3_BUCKET" \
-    $azure_args \
+    ${azure_args} \
     -e SECURITY="$security" \
     -e PYTEST_ARGS="$PYTEST_ARGS" \
-    $FRAMEWORK_ARGS \
+    ${FRAMEWORK_ARGS} \
     -e STUB_UNIVERSE_URL="$STUB_UNIVERSE_URL" \
+    -e PACKAGE_REGISTRY_ENABLED="${package_registry}" \
+    -e PACKAGE_REGISTRY_STUB_URL="${PACKAGE_REGISTRY_STUB_URL}" \
+    -e DCOS_FILES_PATH="$(abs_path "${dcos_files_path}")" \
```

Comment:
That was intentional for couple of reasons: 
- In `sdk_package_registry.py` the business logic varies based upon whether `DCOS_FILES_PATH` is set to a valid path or not.
- There is no harm in doing what I am doing, on the upside, if i pass something like `/Users/myusername/cachefile` as `DCOS_FILES_PATH`, it would be nice to see the same thing in logs as well. And I see no gain by introducing a new string called "/dcos_files" in `test.sh`